Lib/asyncio/controller.py

```diff
@@ -0,0 +1,78 @@
+"""Protocol server controller."""
+
+__all__ = ['Controller']
+
+
+import asyncio
+import os
+import socket
+import threading
+
+try:
+    from socket import socketpair
+except ImportError:
+    from asyncio.windows_utils import socketpair
```

Comment:
No more needed in Python 3.7, just remove it. But i don't know if Controller would only go to asyncio of Python 3.7?